test/unit/services/pgp.service.test.ts

```diff
@@ -17,32 +21,55 @@ describe('Encryption and Decryption', () => {
     const keys = await generateNewKeys();
     const publicKeyInBase64 = keys.publicKeyArmored;
     const message = 'This is a test message';
+    const publicKyberKeyBase64 = keys.publicKyberKeyBase64;
 
     const encryptedMessage = await encryptMessageWithPublicKey({
       message,
       publicKeyInBase64,
+      publicKyberKeyBase64,
     });
 
     expect(encryptedMessage).toBeDefined();
   });
 
+  it('xor should work', async () => {
```

Comment:
It is too generic a test, separate it into different tests in which the description of what it has to achieve is a little more specific

Reply:
Done